posts/Fabio/2018-03-28-hibernate-ogm-5-3-1-Final-released.adoc

```diff
@@ -0,0 +1,118 @@
+= "Hibernate OGM 5.3.1.Final is out"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We're happy to announce the release of:
+http://hibernate.org/ogm/releases/5.3/#get-it[Hibernate OGM 5.3.1.Final]
```

Comment:
Drop the : , it's useless.

An additional sentence in the preamble might be nice. Something like `This is the first maintenance release of the 5.3 branch.`